### PR TITLE
fix: Update readme and cron updates weekly

### PR DESCRIPTION
**Pull request by DickPostma**

### CODE REVIEW

File: scripts/install/README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ The installation script requires some arguments:
 |oidc_secret | Given secret ID|
```
Comment (christianhilbrands) [inline scripts/install/README.md:50]: does the documentation not also have to be change to "oidc_clientsecret"?

**Review by christianhilbrands — CHANGES_REQUESTED** [review]
RfC

File: scripts/install/README.md (hunk 1 above)
Comment (DickPostma) [reply scripts/install/README.md]: Aangepast

**Review by christianhilbrands — APPROVED** [review]
LGTM

**Pull request merged**